### PR TITLE
Numpy Day zero and workshop env file

### PR DESCRIPTION
**Pull request by sosey**
this includes a larger env file with glueviz 0.8.2 and astroquery 0.3.3

### CODE REVIEW

File: jwst-workshop.yml [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+- openpyxl=2.3.2=py35_0
+- openssl=1.0.2j=0
+- pandas=0.18.1=np111py35_0
+- partd=0.3.6=py35_0
+- path.py=8.2.1=py35_0
+- pathlib2=2.1.0=py35_0
+- patsy=0.4.1=py35_0
+- pep8=1.7.0=py35_0
+- pexpect=4.0.1=py35_0
+- photutils=0.2.2=np111py35_1
```
Comment (larrybradley) [inline jwst-workshop.yml]: Photutils needs to be `0.3`.

Comment (larrybradley) [reply jwst-workshop.yml]: It's in AstroConda....

Comment (sosey) [reply jwst-workshop.yml]: ooo, will change, note, this is what's being installed in Joe's script too...

Comment (larrybradley) [reply jwst-workshop.yml]: I just released `photutils 0.3` on Sunday night.

Comment (sosey) [reply jwst-workshop.yml]: right... but I just used joe's install script that mentioned in here to do the full AstroConda install with anaconda and that's the version it pulled. I'm able to update to 0.3 though. I just changed the version in the file, pushing now

**Comment by sosey** [comment]
Merging this now incase I don't get up early enough

**Pull request merged**